developer-cli/Utilities/ProcessHelper.cs

```diff
@@ -123,6 +124,29 @@ public static bool IsProcessRunning(string process)
     {
         return Process.GetProcessesByName(process).Length > 0;
     }
+
+    private static string? FindFullPathFromPath(string command)
+    {
+        Debug.Assert(!string.IsNullOrWhiteSpace(command));
+
+        string[] commandFormats = OperatingSystem.IsWindows() ? ["{0}.exe", "{0}.cmd"] : ["{0}"];
```

Comment:
```suggestion
        var commandFormats = OperatingSystem.IsWindows() ? new[] { "{0}.exe", "{0}.cmd" } : new[] { "{0}" };
```

You can run JetBrains code clean up and it will change `string[]` to `var`.